.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,110 @@
+name: Build and test packages
+
+on:
+  - push
+  - pull_request
```

Comment:
I did this with Eisbuk but it's probably useless: `push` should be enough. My understanding of what is checked out when the target is `pull_request` has changed: I used to think that the result of the PR merge was checked out, but now I believe some manual work would be needed to achieve that (and it wouldn't work in case of conflicts).